ember-app/app/views/issue/reference.js

```diff
@@ -6,7 +7,10 @@ var IssueReferenceView = Ember.View.extend({
   isLoaded: false,
   commit: null,
   commitUrl: Ember.computed.alias("commit.html_url"),
-  message: Ember.computed.alias("commit.commit.message"),
+  processMessage: function(){
+    var message = emojiParser.parse(this.get("commit.commit.message"));
+    this.set("message", message.htmlSafe());
+  }.observes("commit.commit.message"),
```

Comment:
Yah, I think this was a way to handle null commit messages from linked repo commit references (which is no longer an issue), fixing.